.travis.yml

```diff
@@ -1,14 +1,14 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
   - linux
   - osx
 julia:
-  - 0.7
```

Comment:
The package still accepts 0.7, so we should continue to test on it. Adding 1.0 and 1.1 is good though.